### DISCUSSION TITLE
Doris add or modify column

### DISCUSSION THREAD

**Discussion by XinSuiNian** [apache/doris#41947] [A - General / Q&A]
I want to add or modify a field in Doris. I would like to first check if the field exists, and then decide whether to add or modify the field based on that.

like this :  ALTER TABLE if exists test_read_data ADD COLUMN  if not exists aaabbb INT  DEFAULT "0" ;

**Comment by ixzc** [comment]
You can refer this 
https://doris.apache.org/docs/table-design/schema-change/

**Reply by XinSuiNian** [reply]
The content here is not what I'm looking for. I need to check if a field exists before adding or modifying it. If the field exists, then perform the modification; if it does not exist, then do nothing.

I did not find any relevant information in the documentation.
